internal/io/redis/pubsub/redisPub_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2023-2023 EMQ Technologies Co., Ltd.
```

Comment:
The same as above. Any newly added files can copyright to yourself

Reply:
Just changed to my personal email.